chef/cookbooks/hyperv/recipes/register_services_nova.rb

```diff
@@ -0,0 +1,21 @@
+raise if not node[:platform] == 'windows'
+
+powershell "register_services" do
+  code <<-EOH
+    if (-not (Get-Service "#{node[:service][:nova][:name]}" -ErrorAction SilentlyContinue))
+    {
+      # $secpasswd = ConvertTo-SecureString "crowbar" -AsPlainText -Force
+      # $credentials = New-Object System.Management.Automation.PSCredential (".\\Administrator", $secpasswd)
```

Comment:
Heh, sorry: forgot to mention that the comments are also here (as well as in the neutron recipe ;-))